e2e/tests/content-releases/release-details-page.spec.ts

```diff
@@ -0,0 +1,130 @@
+    await submitReleaseButton.click();
+    // See the release the entry was added to
+    await expect(
+      page.getByRole('complementary', { name: 'Releases' }).getByText(releaseName)
+    ).toBeVisible();
+
+    // Publish the release
+    await page.getByRole('link', { name: 'Releases' }).click();
+    await page.getByRole('link', { name: `${releaseName} 8 entries` }).click();
+    await page.getByRole('button', { name: 'Publish' }).click();
+    expect(page.getByRole('heading', { name: releaseName })).toBeVisible();
+    await expect(page.getByRole('button', { name: 'Publish' })).not.toBeVisible();
+    await expect(
+      page.getByRole('button', { name: 'Release edit and delete menu' })
+    ).not.toBeVisible();
+    await expect(page.getByRole('gridcell', { name: 'publish unpublish' })).not.toBeVisible();
+    await expect(
+      page.getByRole('gridcell', { name: 'This entry was published.' }).first()
+    ).toBeVisible();
+  });
+
+  test('A user should be able to edit and delete a release', async ({ page }) => {
+    // Edit the release
+    await page.getByRole('button', { name: 'Release edit and delete menu' }).click();
+    await page.getByRole('button', { name: 'Edit' }).click();
+    await expect(page.getByRole('dialog', { name: 'Edit release' })).toBeVisible();
+    await expect(page.getByRole('button', { name: 'Save' })).toBeDisabled();
+    await page.getByRole('textbox', { name: 'Name' }).fill('Trent Crimm: Independent');
+    await expect(page.getByRole('button', { name: 'Save' })).toBeEnabled();
+    await page.getByRole('button', { name: 'Save' }).click();
+    const editedEntryName = 'Trent Crimm: Independent';
+    await expect(page.getByRole('heading', { name: editedEntryName })).toBeVisible();
+
+    // Delete the release
+    await page.getByRole('button', { name: 'Release edit and delete menu' }).click();
+    await page.getByRole('button', { name: 'Delete' }).click();
+    await page.getByRole('button', { name: 'Confirm' }).click();
+    // Wait for client side redirect to the releases page
+    await page.waitForURL('/admin/plugins/content-releases');
+    await expect(
+      page.getByRole('link', { name: `${editedEntryName} 6 entries` })
+    ).not.toBeVisible();
+  });
+
+  test("A user should be able to change the entry groupings, update an entry's action, remove an entry from a release, and navigate to the entry in the content manager", async ({
+    page,
+  }) => {
+    // Change the entry groupings
+    await expect(page.getByRole('separator', { name: 'Article' })).toBeVisible();
+    await expect(page.getByRole('separator', { name: 'Author' })).toBeVisible();
+    await page.getByLabel('Group by').click();
+    await page.getByRole('option', { name: 'Actions' }).click();
+    await expect(page.getByRole('separator', { name: 'publish', exact: true })).toBeVisible();
+    await expect(page.getByRole('separator', { name: 'unpublish' })).toBeVisible();
```

Comment:
Shall we add a line to check that the items are in the correct separator sections (Publish or Unpublish) ?
And when changing the action, add a line to check that the item moves to the appropriate separator section depending on the action selected ?